src/conf.ml

```diff
@@ -73,7 +73,7 @@ let distros = Distro.(active_distros `X86_64 |> List.filter (fun d ->
 
 let arches_for ~distro = Distro.distro_arches Ocaml_version.Releases.latest distro
 
-let win10_revision : Distro.win10_lcu = `LCU20220913
+let win10_revision : Distro.win10_lcu = `LCU20221011
```

Comment:
Why don't we use `Dockerfile_opam.Distro.win10_current_lcu` here?

Reply:
(maybe I've already asked? can't recall)

Reply:
Looks like a good idea to me.  One less change to make each month.  @dra27 any reason not to do this?

Reply:
For reference it was added as part of https://github.com/ocurrent/docker-base-images/pull/124 which is all David's work but he didn't point to `win10_current_lcu`. I think it's ok to do so.